release.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+set -e
+# avoid issues with relative paths
+cd $(dirname $0)
+
+if [ -z "$(eval "echo \$GH_TOKEN")" ]; then
+  echo "Missing environment variable GH_TOKEN."
+  exit 1
+fi
+
+TEMP_DATA_JSON="release.json"
+PACKAGE="package.json"
+GITHUB_API_ENDPOINT="https://api.github.com/repos/lightninglabs/lightning-app/releases"
+
+if [ $# -ne 1 ]; then
+    echo You must provide only a release version!
+    echo Current release: `sed -n 's/.*"version": "\(.*\)".*/\1/p' ${PACKAGE}`
+    exit 1
+fi
+RELEASE=$1
```

Comment:
I don't fully understand this block. Should the version provided in the package.json be enough to identify the corresponding github release?

Reply:
> No, new version name must be provided as an argument. If you don't provide one the script will exit printing you the current version name for convenience.

Just to clarify, I run `npm version 0.2.2-alpha` locally and then push the git tag and updates package.json to the master. So the package.json already has to new version.

Reply:
Oh, thanks for clearing this up. I'll make the necessary changes. The script will then just take the version from `package.json` and generate a draft release notes to Github.

Reply:
Cool. Sounds good. Thanks